scripts/explain-warn.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+echo 'Creating 10,000 dummy accounts, hang tight...'
+node scripts/populate-session-tokens 10000 2>&1 > /dev/null
+
+SRC="lib/db/mysql.js"
+ACTIVE_PROCEDURES=`grep 'CALL ' "$SRC" | awk -F 'CALL +' '{print $2}' | cut -d '(' -f 1`
+
+for PROCEDURE in $ACTIVE_PROCEDURES
+do
+  FILE=`git grep "$PROCEDURE" | grep 'CREATE PROCEDURE' | cut -d ':' -f 1 | sed '/^$/d'`
+  EXPLAINS=`node scripts/explain-queries "$FILE" "$PROCEDURE"`
+
+  if [ "$EXPLAINS" = "" ]
+  then
+    continue
+  fi
+
+  echo "$EXPLAINS" | while read -r EXPLAIN
+  do
+    EXPLAIN_RESULT=`mysql -D fxa -u root -e "$EXPLAIN" 2>/dev/null`
```

Comment:
This will silently suppress any error connecting to the DB, if may be worth checking `$?` or similar to ensure that we actually got output as expected.